spec/docs/swagger_spec.rb

```diff
@@ -10,10 +10,57 @@
 
   context 'has valid paths' do
     it do
+      expect(subject).to validate(:get, '/manifests/{id}', 200, params)
+    end
+    it do
+      expect(subject).to validate(:get, '/manifests/{id}', 404, unknown_manifest_params)
+    end
+    it do
+      expect(subject).to validate(
+        # TODO must we require 'manifest' key? that is inconsistent with PATCH and /validate
```

Comment:
The challenge with dropping the top level param is that we're gonna have to fight Rails if we do. 

With strong params, we need to explicitly state which param values that we allow. If we allow top-level params, Rails forms will send additional params and we will end up with this error msg:

eg: here is us allowing top level params

```
params.permit(:manifest_attr, :manifest_atr_2)
```

here is the error msg we will get:
```
ActionController::UnpermittedParameters:
       found unpermitted parameters: utf8, token, commit
```

whereas 

```
params.require(:manifest).permit(:manifest_attr_1, :manifest_attr2)
```

makes sure we only look within `manifest` so those other top-level params (utf8, token, commit) are ignored.